doc/reference/reference_lua/box_info.rst

```diff
@@ -31,6 +31,8 @@ variables.
 * **ro** is ``true`` if the instance is in "read-only" mode
   (same as :ref:`read_only <cfg_basic-read_only>` in ``box.cfg{}``),
   or if status is 'orphan'.
+* **ro_reason** is ``nil`` if the instance is in writable mode.
+  When the field is ``true``, it reports an error reason.
```

Comment:
The field is either a nil or a string, so it can't be `true` (only `box.info.ro` can). I would rather say `When the field is not nil, it is a string containing a reason why the instance is read-only`.

Reply:
Updated text